docs/codeql/codeql-cli/about-codeql-workspaces.rst

```diff
@@ -0,0 +1,94 @@
+.. _about-codeql-workspaces:
+
+About CodeQL Workspaces
+=======================
+
+.. include:: ../reusables/beta-note-package-management.rst
+
+CodeQL workspaces are used to group multiple CodeQL packs together. A typical use case for a CodeQL workspace is to develop a set of CodeQL library and query packs that are mutually dependent. For more information on CodeQL packs, see ":doc:`About CodeQL packs <about-codeql-packs>`."
+
+The main benefit of a CodeQL workspace is that it makes it easier for you to develop and maintain multiple CodeQL packs. When you use a CodeQL workspace, all the CodeQL packs in the workspace are available as *source dependencies* for each other when you run a CodeQL command that resolves queries. This makes it easier to develop, maintain, and publish multiple, related CodeQL packs.
+
+In most cases, you should store the CodeQL workspace and the CodeQL packs contained in it in one git repository. This makes it easier to share your CodeQL development environment.
+
+The ``codeql-workspace.yml`` file
+--------------------------------
```

Comment:
It looks as if the underline here is too short, causing the test failure that's blocking the Sphinx docs generation. Will attempt to fix so that I can review the output.

```suggestion
The ``codeql-workspace.yml`` file
----------------------------------
```